service/src/main/java/bio/terra/catalog/schema.json

```diff
@@ -0,0 +1,192 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Catalog Schema V1",
+  "type": "object",
+  "required": ["dct:title", "dct:description", "dct:creator", "dct:issued", "TerraDCAT_ap:hasDataCollection", "dcat:accessURL", "prov:wasGeneratedBy", "storage", "counts", "contributors"],
+  "properties": {
```

Comment:
I don't see `requestAccessURL` in this list of properties, I think it should be there. It should also be in the v0 schema.

Reply:
I was wondering if we should put phsId in here as well, then, but I think we shouldn't because it's stored solely in TDR, and we don't want users adding it here. If we wanted to support phsId for workspaces that would change.
(there's also access level and id, but I think we don't need those because users aren't doing anything with them)